bodyswap.lua

```diff
@@ -35,11 +35,11 @@ Arguments::
 ]====]
 
 if args.help then
- print(usage)
- return
+  print(usage)
+  return
 end
 
-if not dfhack.world.isAdventureMode() then
+if not df.global.gamemode == df.game_mode['ADVENTURE'] then
```

Comment:
```suggestion
if not df.global.gamemode == df.game_mode.ADVENTURE then
```
IMHO is better style